lib/matplotlib/axes/_axes.py

```diff
@@ -2761,11 +2761,11 @@ def errorbar(self, x, y, yerr=None, xerr=None,
             axes, :meth:`set_xlim` or :meth:`set_ylim` must be called
             before :meth:`errorbar`.
 
-          *errorevery*: positive integer or tuple of integers, optional, default:1
-            draws error bars on a subset of the data. *errorevery*=skip draws
-            error bars on the points (x[::skip], y[::skip]).
-            *errorevery*=(skip,shift) draws error bars on the points
-            (x[skip:shift:skip], y[shift:skip:skip]). e.g. errorevery=(6,3)
+        errorevery: positive integer or tuple of integers, optional, default: 1
+            draws error bars on a subset of the data. *errorevery*=N draws
```

Comment:
I switched from (skip, shift) to (start, N) to be consistent with the `set_markevery` documentation.